plugins/mcreator-core/procedures/world_data_isthundering.json

```diff
@@ -1,4 +1,12 @@
 {
+  "args0": [
```

Comment:
There does not seem to be any side checks for this procedure block. Was there some other reason to add this indicator?

Reply:
The method called in the code is overridden by `ClientLevelData` to always return `false`.